lldb/include/lldb/Core/DebuggerEvents.h

```diff
@@ -44,12 +44,16 @@ class ProgressEventData : public EventData {
   uint64_t GetCompleted() const { return m_completed; }
   uint64_t GetTotal() const { return m_total; }
   std::string GetMessage() const {
-    std::string message = m_title;
-    if (!m_details.empty()) {
-      message.append(": ");
-      message.append(m_details);
-    }
-    return message;
+    // Only put the title in the message of the progress create event.
+    if (m_completed == 0) {
```

Comment:
can you create some using/defines for constants like this 0? otherwise it's hard to read